client/src/index.scss

```diff
@@ -38,6 +39,11 @@ main {
     @include minQ(768px) {
         margin-left: 110px;
     }
+    &.unscrollable {
+        width: 100%;
+        //Optional but highly reccomended: enables momentum scrolling on iOS
```

Comment:
This is exactly what we wanted to remove in [this ticket](https://github.com/sanghanet/sanghanet/issues/8). You found a solution for that ticket. Please remove `-webkit-overflow-scrolling: touch` from here, we want our app to stop scrolling when the content ends.

Reply:
Resolved 💯